src/server/session.ts

```diff
@@ -1861,7 +1861,16 @@ namespace ts.server {
                     // Use `hasAction || undefined` to avoid serializing `false`.
                     return { name, kind, kindModifiers, sortText, insertText, replacementSpan: convertedSpan, isSnippet, hasAction: hasAction || undefined, source, sourceDisplay, isRecommended, isPackageJsonImport, isImportStatementCompletion, data };
                 }
-            }), (a, b) => compareStringsCaseSensitiveUI(a.name, b.name));
+            }), (a, b) => {

```

Comment:
This is kind of a weird place for sorting to happen, but this has always been the only place we’ve sorted completion items, and moving it to somewhere more logical would have to update a ton of fourslash tests.